client/src/index.js

```diff
@@ -86,6 +92,8 @@ const Routes = () => {
           {isLoggedIn() ? (
             <Route path="/add-document/" component={AddDocumentForm} />
           ) : null}
+
+          <Route path="/dashboard" component={Dashboard} />
```

Comment:
The dashboard route should only be active if the user is logged in.

Reply:
Changed